site/content/agentic-ai-suite/graph-analytics/api.md

```
@@ -547,6 +547,170 @@ curl -H "Authorization: bearer $ARANGO_GRAPH_TOKEN" -XPOST -d '{"database":"_sys
+  `I64`, `F64`, `JSON`).
+- `edge_attributes` (array, optional): A list of edge attributes to load, with
+  the same structure as `vertex_attributes`.
+- `batch_size` (integer, optional): The number of documents per batch
+  (default: `400000`).
+
+The response contains a `job_id` and `graph_id`. Use the `job_id` to track the
+loading progress via the [Jobs API](#list-all-jobs). The loading job reports a total
+of `2` progress steps: `1` after all vertices have been processed, and `2` when
+all edges are processed and the graph is ready.
+
+**Example using a graph traversal (single-phase load):**
```

markdownlint-cli2 (0.22.0)

[warning] Strong style
Expected: underscore; Actual: asterisk

(MD050, strong-style)